.github/workflows/ci.yaml

```diff
@@ -28,6 +28,10 @@ jobs:
             ROS_DISTRO: humble
           - IMAGE: humble-ci-testing
             ROS_DISTRO: humble
+          - IMAGE: jazzy-ci
+            ROS_DISTRO: jazzy
+          - IMAGE: jazzy-ci-testing
+            ROS_DISTRO: jazzy
```

Comment:
Note that `ci` and `ci-testing` are identical since #2793: Because `osrf/ros2:testing` (the base image of `ci-testing`) was broken upstream at the time, I disabled building it and aliased it to `ci`.
```suggestion
```